docs/pages/Builder.md

```diff
@@ -81,6 +81,7 @@ All available standard tasks are documented [in the API reference](https://sap.g
 ^2^ Enabled for projects defining a [component preload configuration](./Configuration.md#component-preload-generation)  
 ^3^ Enabled in `self-contained` build, which disables `generateComponentPreload` and `generateLibraryPreload`  
 ^4^ Enabled for projects defining a [bundle configuration](./Configuration.md#custom-bundling)  
+^5^ Can be enabled for framework projects via `includeTask` option. For other projects the task is always skipped  
```

Comment:
^5^ Can be enabled for framework projects via the `includeTask` option. For other projects, this task is skipped  

(and below)

Reply:
Thanks!